internal/server/iigointernal/executive.go

```diff
@@ -162,12 +162,12 @@ func (e *executive) reset(val string) {
 //requestRuleProposal asks each island alive for its rule proposal.
 func (e *executive) requestRuleProposal() {
 	e.budget -= serviceCharge
-	var rules []string
+	var ruleProposals []string
 	for _, island := range getIslandAlive() {
-		rules = append(rules, iigoClients[shared.ClientID(int(island))].RuleProposal())
+		ruleProposals = append(ruleProposals, iigoClients[shared.ClientID(int(island))].RuleProposal())
```

Comment:
what happens if the proposed rule isn't a valid one. Is this checked later on?

Reply:
Not for now, considering the changes that will likely come with the invented rules a system as described would need to be implemented, but until it's in I'm unsure what value we'd get out of it.

Reply:
Screw it I added one anyway